include/CFL/CFLGraphBuilder.h

```diff
@@ -83,22 +101,22 @@ class CFLGraphBuilder
                 {
                     NormalGepCGEdge *nGepEdge = SVFUtil::dyn_cast<NormalGepCGEdge>(edge);
                     CFLGrammar::Attribute offset =  nGepEdge->getConstantFieldIdx();
-                    cflGraph->addAttribute(edgeLabel, offset);
+                    addAttribute(edgeLabel, offset);
                     edgeLabel = CFLGrammar::getAttributedKind(offset, edgeLabel);
                     cflGraph->addCFLEdge(cflGraph->getGNode(edge->getSrcID()), cflGraph->getGNode(edge->getDstID()), edgeLabel);
-                    std::string key = cflGraph->sym2LabelMap[edge->getEdgeKind()];
+                    std::string key = kind2LabelMap[edge->getEdgeKind()];
                     key.pop_back();
                     key.pop_back();    // _i standsfor attribute variable should place at last
                     key.append("bar_i");   // for example Gep_i should be Gepbar_i, not Gep_ibar
-                    cflGraph->addCFLEdge(cflGraph->getGNode(edge->getDstID()), cflGraph->getGNode(edge->getSrcID()), CFLGrammar::getAttributedKind(offset, cflGraph->label2SymMap[key]));
```

Comment:
Yes, "bar_i" is for every Kind with attribute, so currently if that come with two different attribute let's say "i" and "j", then it will just treat as "i" as attribute, but "j" will be just a new part of Nonterminal.
I think I will also need to support multiple attribute, in the case of that kind of grammar input.